src/frontend/src/app.d.ts

```diff
@@ -24,6 +24,7 @@ declare namespace svelteHTML {
 		'on:oisyCkEthMinterInfoStatus'?: (event: CustomEvent<any>) => void;
 		'on:oisyCkEthereumPendingTransactions'?: (event: CustomEvent<any>) => void;
 		'on:oisyValidateAddresses'?: (event: CustomEvent<any>) => void;
+		'on:oisyRefreshUserProfile'?: (event: CustomEvent<any>) => void;
```

Comment:
Now that we have removed the `emit`, this is still required`? Likewise for the `svelte:window` related listener?

Reply:
it will be for sure for the next PR: do you prefer I add it in the next one?

Reply:
No I mean shouldn't it be removed - not added no? This PRs adds it but, it's not needed anymore no?

Reply:
indeed! removed! i will add it back when needed